doc/doxygen/headers/matrixfree.h

```diff
@@ -81,12 +85,8 @@ digraph G
   subgraph misclibs {
   systems    [label="Operators"];
   }
-  simd        -> fevalues [dir="none", color="transparent"];
-  fevalues    -> mf       [dir="none", color="transparent"];
-  mf          -> cuda     [dir="none", color="transparent"];
-  cuda        -> tbb      [dir="none", color="transparent"];
   opencascade -> manifold [dir="none"];
-  gmg         -> solvers  [dir="none", style="dashed", color="black"];
+  gmg -> solvers          [dir="none", style="dashed", color="black"];
```

Comment:
Remove this line

Reply:
Thanks for the fix and the offline tests! I am not completely happy with the graph since the nodes `Triangulation`, `FiniteElement`, `Mapping`, and `Quadrature` are mirrored in comparison to the matrix-based path. But I guess this is fine for now to be merged. 

I will install a different version of dot and try to find a solution which works. My version is:
```
> dot -V
dot - graphviz version 2.30.1 (20180420.1509)
```